vyxal/structure.py

```diff
@@ -113,6 +113,11 @@ def __init__(self, modifier: str, *branches: Branch):
         self.modifier = modifier
         self.function_A = branches[0]
 
+    def __repr__(self):
```

Comment:
We can probably delete the `__repr__`s now.

Reply:
No.

Reply:
The parsing test cases use string equality, so these might still be needed.